criu/cr-restore.c

```diff
@@ -3235,7 +3238,18 @@ rst_prep_creds_args(CredsEntry *ce, unsigned long *prev_pos)
 	memcpy(args->cap_prm, ce->cap_prm, sizeof(args->cap_prm));
 	memcpy(args->cap_bnd, ce->cap_bnd, sizeof(args->cap_bnd));
 
-	if (ce->n_groups) {
+	n = getgroups(0, NULL);
```

Comment:
Btw, Andrei had another comment that I think we haven't addressed yet. 
"you don't need to call sys_getgroups if n isn't equal to ce->n_groups"